script.rb

```diff
@@ -8,6 +8,9 @@
 
 DatabaseClient.all.each do |twitter_account|
   next if twitter_account['name'] == 'Playerapp_cbsk'
+  # I remove Player_twi because this account has too much data.
+  # But I want to save cookies data, so deal with it by adding next process.
+  next if twitter_account['name'] == 'Player_twi'
```

Comment:
It would be better if we can handle these via environment variable.

Reply:
@yusuko ↑

Reply:
Yes, that's true.
Thank you!